config/config.php

```diff
@@ -73,14 +76,23 @@ public function getInstallSql()
                 INSERT INTO `[prefix]_checkout_currencies` (`id`, `name`) VALUES (6, "CHF");';
     }
 
-    public function getUpdate($installedVersion)
+    public function getUpdate(string $installedVersion): string
     {
         switch ($installedVersion) {
             case "1.0":
             case "1.1":
                 // Convert tables to new character set and collate
                 $this->db()->query('ALTER TABLE `[prefix]_checkoutbasic` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;');
                 $this->db()->query('ALTER TABLE `[prefix]_checkoutbasic_currencies` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;');
+            // no break
+            case "1.2.0":
+            case "1.3.0":
+            case "1.4.0":
+            case "1.4.1":
```

Comment:
Where did you see a version 1.4.1 and 1.4.2 of the checkout module? You just updated the version number above from 1.4.0 to 1.5.0.

Reply:
kommt hier her: https://github.com/IlchCMS/Ilch-2.0/blob/b0356b91b4852e7d871828332f71b1d6f8d01078/application/modules/checkoutbasic/config/config.php#L90

Reply:
Sind aber "unterschiedliche" Module. Einmal den Key "checkout" und einmal "checkoutbasic". Also wenn es bei diesem Modul nicht wirklich eine Version 1.4.1 und 1.4.2 gab, dann bräuchte man das hier nicht eintragen. Würde aber auch nichts kaputt machen.